plugin/forward/setup.go

```diff
@@ -41,6 +35,14 @@ func setup(c *caddy.Controller) error {
 	c.OnStartup(func() error {
 		return f.OnStartup()
 	})
+	c.OnStartup(func() error {
```

Comment:
lets also mention `c.OnStartup()` in README example

Reply:
good point. done